go.mod

```diff
@@ -48,8 +49,11 @@ require (
 	k8s.io/apiserver v0.18.0
 	k8s.io/client-go v0.18.0
 	k8s.io/cloud-provider v0.18.0
+	k8s.io/code-generator v0.18.1 // indirect
 	k8s.io/component-base v0.18.0
+	k8s.io/gengo v0.0.0-20200413195148-3a45101e95ac // indirect
 	k8s.io/klog v1.0.0
```

Comment:
no need of klog anymore?

Reply:
others might still need it?

Reply:
not required I suppose , if you remove it and run go mod tidy , it would be added automatically as indirect dependency if required.